src/RoslynInsertionTool/scripts/UpdateExistingPr.ps1

```diff
@@ -16,7 +16,8 @@ param([string] $enlistmentPath,
       [string] $updateAssemblyVersions,
       [string] $updateCoreXTLibraries,
       [string] $visualStudioBranchName,
-      [string] $writePullRequest)
```

Comment:
Just a note to remember to update the Azure DevOps release definition when this is in to take into account the script name change.  I can help with this if you need.

Reply:
yes, that would be helpful.

Reply:
Synced up offline. Will make a change.